src/Controller/Component/RatingComponent.php

```diff
@@ -108,7 +108,7 @@ public function rate($rate, $rating, $user, $redirect = false) {
 			$Model = $Controller->{$this->getConfig('modelName')};
 			$newRating = $Model->saveRating($rate, $user, $rating);
 			if ($newRating) {
-				$rating = round($newRating->newRating);
+				$rating = round($newRating);
```

Comment:
I changed it to $newRating->rating, as it is apparently an entity now.
See master code.